Guardian.Backend/Guardian.Service/Features/CustomerFeatures/Queries/GetAllGamesQuery.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Guardian.Domain.Entities;
+using Guardian.Persistence;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Guardian.Domain.Entities;
+
+namespace Guardian.Service.Features.CustomerFeatures.Queries
+{
+    public class GetAllGamesQuery : IRequest<IEnumerable<Game>>
+    {       
+        PagiantionParams pagiantionParams = new PagiantionParams();
```

Comment:
my to chcemy przekazywać z kontrolera przez parametr zamiast tworzyć nowąinstancje tutaj